Sources/Services/ContainerSandboxService/SandboxService.swift

```diff
@@ -866,25 +913,36 @@ public actor SandboxService {
         return container
     }
 
-    private func gracefulStopContainer(_ lc: LinuxContainer, stopOpts: ContainerStopOptions) async throws {
```

Comment:
Note about code not modified in this PR:

```swift
private nonisolated func configureProcessConfig()
```

This should be able to become:

```swift
private static func configureProcessConfig()
```

Same for `closeHandle()`.

Reply:
Another lowkey obsessive nit in the non-modified code above - move `getDefaultNameserver` out from between the `configure...()` funcs. All the private methods could probably stand to be reordered sensibly. In a swift file that's pushing 1200 lines it will help new developers.